tests/attacks/property_inference_test.py

```diff
@@ -8,29 +8,32 @@
 
 
 def test_property_inference_attack():
-    train_dataset, test_dataset = dataset_downloader()
-    input_shape =  test_dataset[0].shape
-    num_classes = 2
-    num_elements_per_classes = {0: 1000, 1: 1000}
-   
+    train_dataset, test_dataset = dataset_downloader("MNIST")
+    input_shape = test_dataset[0][0].shape
+
+    num_elements_per_class = {0: 500, 1: 500} #TODO change back to higher number
+    num_classes = len(num_elements_per_class)
+
     train_set = new_dataset_from_size_dict(
-            train_dataset, num_elements_per_classes
+        train_dataset, num_elements_per_class
     )
     test_set = new_dataset_from_size_dict(
-            test_dataset, num_elements_per_classes
+        test_dataset, num_elements_per_class
     )
 
-    model = FCNeuralNet()
+    model = FCNeuralNet(input_shape)
     trainer(
-        train_set, test_set, num_elements_per_classes, model
+        test_set, num_elements_per_class, model
     )
 
     # change pytorch classifier to art classifier
     target_model = Classifier._to_art_classifier(model, num_classes, input_shape)
 
-    attack = PropertyInferenceAttack(target_model, test_dataset)
-    attack.attack()
-    """res = attack.attack()
-    f = open('test.txt', 'a')
-    f.write(str(res))
-    f.close()"""
+    attack = PropertyInferenceAttack(target_model, train_dataset)
+    print(attack.attack())
```

Comment:
What is the purpose of the test? How would it fail and when? What would be the thing that a user or developer could understand from it? In its current form, the test seems not really meaningful to me.

Reply:
We based this on the other tests in the test files. But you are absolutely right, our test is not a real test - I added it as a subject for the sprint review.